config/install/editor.editor.basic_html.yml

```diff
@@ -39,9 +39,7 @@ settings:
           name: Tools
```

Comment:
Should we delete this file?

Reply:
> Should we delete this file?

I think that's probably a good idea but it might be better to do separate from the 9.5.x update.